src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class StringTest {
+
+    @Test
+    void srs1Test() {
+        String value1 = "1,2";
+        assertThat(value1.split(",")).contains("1");
+        assertThat(value1.split(",")).contains("2");
+        assertThat(value1.split(",")).containsExactly("1", "2");
```

Comment:
```suggestion
        assertThat(value1.split(",")).containsExactly("1", "2");
```
마지막 하나로 모두 검증할 수 있지 않을까?
테스트 코드 또한 유지보수해야하기 때문에 최대한 간결하게 작성해 보는 연습을 하면 어떨까?